backend/Cargo.toml

```diff
@@ -29,19 +30,23 @@ error-stack = "0.4"
 thiserror = "1.0"
 
 ## General
-tokio = { version = "1.32", features = ["rt", "macros", "rt-multi-thread"] }
+tokio = { version = "1.32", features = ["rt", "macros", "rt-multi-thread" ] }
 hyper = "0.14"
+lazy_static = "1.4"
 
 ## OpenAPI + Swagger
-utoipa = { version = "3.4", features = ["axum_extras",  "chrono", "openapi_extensions"], optional = true }
-utoipa-swagger-ui = { version = "3.1", features = ["axum"], optional = true }
-utoipa-redoc = { version = "0.1", features = ["axum"], optional = true }
-utoipa-rapidoc = { version = "0.1", features = ["axum"], optional = true }
+utoipa = { version = "4.0", features = ["yaml", "axum_extras",  "chrono", "openapi_extensions"], optional = true }
+utoipa-swagger-ui = { version = "4.0", features = ["axum"], optional = true }
+utoipa-redoc = { version = "1.0", features = ["axum"], optional = true }
+utoipa-rapidoc = { version = "1.0", features = ["axum"], optional = true }
 
 ## CLI
 cli = { path = "../cli" }
 
-[features]
-default = [ "swagger" ]
-swagger = [ "utoipa", "utoipa-swagger-ui" , "utoipa-redoc", "utoipa-rapidoc" ]
+## Frontend
+frontend = { path = "../frontend", optional = true }
```

Comment:
It looks like the dependency on the frontend is unnecessary. Right now they can be built independently

Reply:
No, without that dependency, the frontend won't be built with `cargo run` command

Reply:
I think, that this problem can be resolved by placing `frontend` before `backend` in members list inside top level `Cargo.toml` (https://github.com/qoollo/bob-management/blob/main/Cargo.toml#L10-L11)